.circleci/config.yml

```diff
@@ -151,6 +151,13 @@ commands:
                     fi
                   fi
 
+  validate-generated-code:
+    steps:
+      - run:
+          name: Trigger generated code validation
+          command: |
+            pip3 install requests && python3 scripts/ci-circleci-start-pipeline.py --token ${MOBILE_METRICS_TOKEN} --origin-slug mapbox/mapbox-maps-android --target-slug mapbox/mapbox-maps-stylegen
```

Comment:
Is `MOBILE_METRICS_TOKEN` correct here?

Reply:
This is a circle CI API token, we can reuse this token to trigger other pipelines.